client/pages/BikeInfo.tsx

```diff
@@ -31,7 +31,7 @@ const BikeInfo = ({route, navigation}) => {
                 </View>
                 <Text style={styles.bikeDescriptionText}>{bike.description}</Text>
                 <View style={styles.buttonContainer}>
-                    <TouchableOpacity onPress={() => navigation.navigate('Return Bike', {bike: bike})} style={styles.button}>
+                    <TouchableOpacity onPress={() => navigation.navigate('Booking', { screen: 'Return Bike' }, {bike: bike})} style={styles.button}>
```

Comment:
Thanks for sharing how you did this in your PR comment...I was trying to figure out how to do this the other day.